Pipfile

```diff
@@ -13,7 +13,7 @@ django-ajax-selects = "==1.7.1"
 django-environ = "*"
 django-filter = "==1.1"
 django-storages = "*"
-djangorestframework = "==3.5.3"
+djangorestframework = "==3.9.4"
```

Comment:
I'm not sure about that. 

`rest_framework.filters.DjangoFilterBackend` changed  as `django_filters.rest_framework.DjangoFilterBackend` in version 3.6

This code call rest_framework.

https://github.com/mozilla/network-pulse-api/blob/43bdf69b301159eb95e29e37076cc8ef6331c89f/pulseapi/creators/views.py#L73-L74

Did you check for deprecation warnings?

`python -Wd manage.py test`